src/test/kotlin/com/igorwojda/integer/stepsgenerator/challenge.kt

```diff
@@ -4,7 +4,15 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 fun generateSteps(n: Int): List<String> {
-    TODO("not implemented")
+    val steps = MutableList(n) { "" }
+    (0 until n).forEach { i ->
+        (0 until n).forEach { j ->
+            if (j < i + 1) {
+                steps[i] += "#"
+            } else steps[i] += " "
```

Comment:
inconsistent formatting. Please use curly braces in if-else statements even if they are single line expressions. It makes it easier to read & debug.